apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -364,7 +364,19 @@ const FundIssues = ({ issues, mode }) => {
     if (bountySettings.fundingModel === 'Fixed') return appState.tokens
     return [appState.tokens.find(t => t.addr === bountySettings.bountyCurrency)]
   }, [bountySettings])
-  const [ bounties, setBounties ] = useState(bountiesFor({ bountySettings, issues, tokens }))
+  const bountylessIssues = useMemo(
+    () => issues.filter(i => !i.hasBounty),
+    [issues]
+  )
+  const alreadyAdded = useMemo(
+    () => issues.filter(i => i.hasBounty),
+    [issues]
+  )
```

Comment:
@PeterMPhillips while resolving conflicts, I changed this logic. Rather than loop through `issues` once, on every render pass, it will now loop through `issues` twice on initial render, and then never again, unless the `issues` prop changes (which should only happen if you close the panel and re-open it).